tests/test_conditional_flow_matcher.py

```diff
@@ -115,13 +123,14 @@ def test_fm(method, sigma, shape):
         x0, x1 = sample_plan(method, x0, x1, sigma)
 
     torch.manual_seed(TEST_SEED)
+    if test_eps:
+        # compute to get same t seed
+        eps = torch.randn_like(x0)
     t_given_init = torch.rand(batch_size)
     t_given = t_given_init.reshape(-1, *([1] * (x0.dim() - 1)))
     sigma_pad = pad_t_like_x(sigma, x0)
-    epsilon = torch.randn_like(x0)
-    computed_xt, computed_ut = compute_xt_ut(method, x0, x1, t_given, sigma_pad, epsilon)
+    computed_xt, computed_ut = compute_xt_ut(method, x0, x1, t_given, sigma_pad, ret_eps)
 
     assert torch.all(ut.eq(computed_ut))
-    assert torch.all(xt.eq(computed_xt))
-    assert torch.all(eps.eq(epsilon))
+    assert torch.allclose(xt, computed_xt)
```

Comment:
why did you change this exactly? Safer to use eq no?

Reply:
should prefer `allclose` for all float comparisons. Should only use `eq` for integers. Probably should change them all honestly.